fairroot/base/event/FairTimeStamp.h

```diff
@@ -75,6 +75,7 @@ class FairTimeStamp : public FairMultiLinkedData_Interface
     }
 
     virtual bool operator<(const FairTimeStamp* rValue) const { return GetTimeStamp() < rValue->GetTimeStamp(); }
+    bool operator<(const FairTimeStamp& rhs) const { return GetTimeStamp() < rhs.GetTimeStamp(); }
```

Comment:
Why those getters and not accessing the member variable directly? It is in this exact same class (so it would even work, if it was private)?

Reply:
See my answer to @karabowi comment. I will remove these changes from here.

Reply:
done